Formula/m/mono.rb

```diff
@@ -1,8 +1,7 @@
 class Mono < Formula
   desc "Cross platform, open source .NET development framework"
   homepage "https://www.mono-project.com/"
-  url "https://download.mono-project.com/sources/mono/mono-6.12.0.199.tar.xz"
-  sha256 "c0850d545353a6ba2238d45f0914490c6a14a0017f151d3905b558f033478ef5"
+  url "https://github.com/mono/mono.git", using: :git, tag: "mono-6.12.0.206", revision: "0cbf0e290c31adb476f9de0fa44b1d8829affa40"
```

Comment:
does upstream stop publishing artifact into `https://download.mono-project.com/sources/mono`?

Reply:
Not every patch version is released as tarball. 

| Version | Date |
| ------- | ------|
| mono-6.12.0.90 | 2020-09-03 |
| mono-6.12.0.107 | 2020-12-09 |
| mono-6.12.0.122 | 2021-02-22 |
| mono-6.12.0.182 | 2022-06-14 |
| mono-6.12.0.199 | 2023-07-27 |

Thus we need to download from git tag like [arch linux package does](https://gitlab.archlinux.org/archlinux/packaging/packages/mono/-/blob/main/PKGBUILD?ref_type=heads).